### PR TITLE
First Pass Rough Copyediting Updates (June 2023)

### PR DESCRIPTION
**Pull request by OneRadRedDad**
# Pull request

**Purpose**
Copyediting changes--my first foray into Github PRs and MkDocs (a two-for-one special!)

As I get more into this, I'm starting to see some extra fine-tuning this guide could benefit from at a later date. This PR will serve as a first pass for some rough copyediting updates, adding clarification and smoothing out a few rough edges where seen.

Later PR may be created to unify the guides into a more strict style guide.

### CODE REVIEW

**Comment by TRaSH-** [comment]
Thnx for these updates
I want to review them before I merge them
Hopefully before the weekend

File: docs/Hardlinks/How-to-setup-for/Docker.md
Comment (bakerboy448) [inline docs/Hardlinks/How-to-setup-for/Docker.md:13]: @TRaSH- is this a specific keyword at all?

File: includes/hardlinks/breakdown-folder-structure-docker.md
Comment (bakerboy448) [inline includes/hardlinks/breakdown-folder-structure-docker.md:15]:
```suggestion
Abbreviated Docker-compose volumes example:
```

etc.

File: includes/hardlinks/breakdown-folder-structure-docker.md
Comment (bakerboy448) [inline includes/hardlinks/breakdown-folder-structure-docker.md:37]:
```suggestion
Abbreviated Docker-compose volumes example
```

File: includes/hardlinks/breakdown-folder-structure-docker.md
Comment (bakerboy448) [inline includes/hardlinks/breakdown-folder-structure-docker.md:59]:
```suggestion
Abbreviated Docker-compose volumes example
```

File: includes/hardlinks/breakdown-folder-structure-docker.md
Comment (bakerboy448) [inline includes/hardlinks/breakdown-folder-structure-docker.md:81]:
```suggestion
Abbreviated Docker-compose volumes example
```

File: includes/hardlinks/folder-structure.md
Comment (bakerboy448) [inline includes/hardlinks/folder-structure.md:7]:
```suggestion
    the only thing you should avoid is `/home` (because folders in `/home`  do have restrictive permissions by default).
```

**Review by bakerboy448 — CHANGES_REQUESTED** [review]
LGTM with a few nitpicks - pending trash's opinion on the note => notes for mkdcos

**Comment by bakerboy448** [comment]
switch it from draft once you're fully done with any other edits and then we should only need the `notes` vs. `note` opinion from trash

thanks for the updates!

File: includes/hardlinks/folder-structure.md
Comment (OneRadRedDad) [reply includes/hardlinks/folder-structure.md]: Updated this one

File: includes/hardlinks/breakdown-folder-structure-docker.md
Comment (TRaSH-) [reply includes/hardlinks/breakdown-folder-structure-docker.md]: > 

I think the word `Abbreviated` is to complicated for the non native English speaking people.

File: includes/hardlinks/breakdown-folder-structure-docker.md
Comment (bakerboy448) [reply includes/hardlinks/breakdown-folder-structure-docker.md]: twas an oops

File: includes/hardlinks/breakdown-folder-structure-docker.md
Comment (OneRadRedDad) [reply includes/hardlinks/breakdown-folder-structure-docker.md:15]: Added :)

**Comment by TRaSH-** [comment]
> LGTM with a few nitpicks - pending trash's opinion on the note => notes for mkdcos

https://squidfunk.github.io/mkdocs-material/reference/admonitions/

File: docs/Hardlinks/How-to-setup-for/Docker.md
Comment (TRaSH-) [reply docs/Hardlinks/How-to-setup-for/Docker.md:13]: https://squidfunk.github.io/mkdocs-material/reference/admonitions/

**Comment by OneRadRedDad** [comment]
> > LGTM with a few nitpicks - pending trash's opinion on the note => notes for mkdcos
> 
> https://squidfunk.github.io/mkdocs-material/reference/admonitions/

Ahh. I'll fix that next. Working on permissions.md right now.

File: includes/hardlinks/breakdown-folder-structure-docker.md
Comment (OneRadRedDad) [reply includes/hardlinks/breakdown-folder-structure-docker.md]: How about "Docker-compose Volumes Example Snippet" ?

Comment (OneRadRedDad) [reply includes/hardlinks/breakdown-folder-structure-docker.md]: Or Example Docker-compose Volumes Excerpt?

**Comment by bakerboy448** [comment]
bazarr conflicted

also not to add more work, but would you be able to implement and fix  - as best as possible (i.e. at least the md section)
https://github.com/TRaSH-Guides/Guides/issues/1404

**Comment by bakerboy448** [comment]
@nemchik FYI you guys probably want to coordinate on editor config

File: docs/Bazarr/Setup-Guide.md
Comment (nemchik) [inline docs/Bazarr/Setup-Guide.md:12]: why the 4th dash?

File: docs/Bazarr/Setup-Guide.md
Comment (nemchik) [inline docs/Bazarr/Setup-Guide.md:104]: why the 4th dash?

File: docs/Bazarr/Setup-Guide.md
Comment (nemchik) [inline docs/Bazarr/Setup-Guide.md:192]: why the 4th dash?

File: docs/Bazarr/Setup-Guide.md
Comment (nemchik) [inline docs/Bazarr/Setup-Guide.md:251]: why the 4th dash?

File: docs/Bazarr/Setup-Guide.md
Comment (nemchik) [inline docs/Bazarr/Setup-Guide.md:324]: why the 4th dash?

File: docs/Bazarr/Setup-Guide.md
Comment (nemchik) [inline docs/Bazarr/Setup-Guide.md:334]: why the 4th dash?

File: includes/hardlinks/breakdown-folder-structure-docker.md
Comment (nemchik) [inline includes/hardlinks/breakdown-folder-structure-docker.md:15]:
```suggestion
Docker-compose Volumes Example Snippet:
```

File: includes/hardlinks/breakdown-folder-structure-docker.md
Comment (nemchik) [inline includes/hardlinks/breakdown-folder-structure-docker.md:37]:
```suggestion
Docker-compose Volumes Example Snippet:
```

File: includes/hardlinks/breakdown-folder-structure-docker.md
Comment (nemchik) [inline includes/hardlinks/breakdown-folder-structure-docker.md:59]:
```suggestion
Docker-compose Volumes Example Snippet:
```

File: includes/hardlinks/breakdown-folder-structure-docker.md
Comment (nemchik) [inline includes/hardlinks/breakdown-folder-structure-docker.md:81]:
```suggestion
Docker-compose Volumes Example Snippet:
```

File: includes/hardlinks/folder-structure.md
Comment (nemchik) [inline includes/hardlinks/folder-structure.md:15]: Why not use `!!! note` ?

**Review by nemchik — CHANGES_REQUESTED** [review]
Left a number of comments, but also as an aside, you probably want to locally test this, especially when indentation is being changed in some sections. mkdocs may not render things as expected if the indentation is off.

**Comment by bakerboy448** [comment]
needs a rebase

**Comment by TRaSH-** [comment]
so many grammar fixes :(

**Comment by zakkarry** [comment]
Not sure if this has been abandoned or not, but as @bakerboy448 said definitely needs a rebase.

As this is in line with some of the work I've adopted (proofing) - I'd be willing to join the effort if this is picked up, although I will say I dont really want to be the one to rebase and address the conflicts that have cropped up.

It might be a better road to address this in a series of section-specific PR's, since this is rather lengthy and massive undertaking broadly. It would seem to me that if this is done this way, frequent rebasing and conflict resolution and reproofing will need to be done throughout.

Does not sound like a fun undertaking to do that way.

**Comment by OneRadRedDad** [comment]
This PR has gotten very outdated after life got hectic. I've since gotten better at markdown and github. I'll close this PR and start over, making new ones with much smaller changes at a time.

**Comment by zakkarry** [comment]
@OneRadRedDad Jump on the discord and shoot me a DM or tag me if you want, I've been proofing most PR's for the last couple of months and would be down to make a plan if you're going to tackle things systematically.

**Pull request closed without merging**
